x-pack/plugins/apm/server/saved_objects/apm_service_groups.ts

```diff
@@ -7,8 +7,37 @@
 
 import { SavedObjectsType } from '@kbn/core/server';
 import { i18n } from '@kbn/i18n';
+import type { SavedObjectMigrationFn } from '@kbn/core/server';
 import { APM_SERVICE_GROUP_SAVED_OBJECT_TYPE } from '../../common/service_groups';
 
+interface ApmServiceGroupsPre850 {
+  groupName: string;
+  kuery: string;
+  description: string;
+  serviceNames: string[];
+  color: string;
+}
+
+interface ApmServiceGroups {
+  groupName: string;
+  kuery: string;
+  description: string;
+  color: string;
+}
+
+const migrateApmServiceGroups850: SavedObjectMigrationFn<
```

Comment:
Is that a convention to put the version in the end like that? Perhaps we can make it a little clearer:
```suggestion
const migrateApmServiceGroups_v8_5_0: SavedObjectMigrationFn<
```

Reply:
Not sure if this is a convention but It's aligned with https://github.com/elastic/sdh-kibana/issues/3211